docs/index.adoc

```diff
@@ -0,0 +1,125 @@
+
+The steps are the following:
+
+1. Create `Dockerfile` with following content:
+
+        FROM quay.io/wildfly/wildfly
+        ADD your-awesome-app.war /opt/jboss/wildfly/standalone/deployments/
+
+2. Place your `your-awesome-app.war` file in the same directory as your `Dockerfile`.
+3. Run the build with `podman build --tag=wildfly-app .`
+4. Run the container with `podman run wildfly-app`. Application will be deployed on the container boot.
+
+== Logging
+
+You can enable loggers by executing WildFly CLI script/commands when building the image. For example create a `Dockerfile` with the following content:
+
+    FROM quay.io/wildfly/wildfly
+    RUN $JBOSS_HOME/bin/jboss-cli.sh --commands="embed-server,/subsystem=logging/console-handler=CONSOLE:write-attribute(name=level,value=TRACE),/subsystem=logging/logger=org.wildfly.security:add(level=TRACE)"
+    # Delete any content generated during embedded execution. Required to avoid read-only directory at server startup
+    RUN rm -rf $JBOSS_HOME/standalone/configuration/standalone_xml_history; rm -rf $JBOSS_HOME/standalone/tmp; rm -rf $JBOSS_HOME/standalone/data
+    CMD ["/opt/jboss/wildfly/bin/standalone.sh", "-b", "0.0.0.0", "-bmanagement", "0.0.0.0"]
+
+Then you can build the image:
+
+    podman build --tag=wildfly-logging .
+
+Run it with:
+
+    podman run -p 9990:9990 wildfly-logging
+
+In another terminal, attempt to access the management console `curl 127.0.0.1:9990/management`, you will see security traces displayed in the console.
+
+== Extending the image with the management console
+
+To be able to create an admin user to access the management console first create a shell script named `startup.sh` to add a user and start the server:
+
+    #!/bin/bash
+    $JBOSS_HOME/bin/add-user.sh -u ${ADMIN_USER} -p ${ADMIN_PASSWORD} --silent
+    $JBOSS_HOME/bin/standalone.sh -b 0.0.0.0 -bmanagement 0.0.0.0
+
+Then create a `Dockerfile` with the following content:
+
+    FROM quay.io/wildfly/wildfly
+    COPY --chown=jboss:jboss startup.sh $JBOSS_HOME/bin
+    RUN chmod a+x $JBOSS_HOME/bin/startup.sh
+    CMD ["/opt/jboss/wildfly/bin/startup.sh"]
+
+Then you can build the image:
+
+    podman build --tag=wildfly-admin .
+
+Run it with:
+
+    podman run -p 9990:9990 --env=ADMIN_USER=alice --env=ADMIN_PASSWORD=Admin#70365 -it wildfly-admin
+
+Management console will be available on the port `9990` of the container and you can connect with `alice` : `Admin#70365`.
+
+== Image internals
+
+This image extends the link:https://hub.docker.com/_/eclipse-temurin[`eclipse-temurin`] JDK. This base OS used is link:https://catalog.redhat.com/software/containers/ubi9-minimal/61832888c0d15aff4912fe0d[`ubi9-minimal`].
+
+The server is run as the `jboss` user which has the uid/gid set to `1000`.
+
+WildFly is installed in the `/opt/jboss/wildfly` directory.
```

Comment:
maybe mention that the `JBOSS_HOME` env var can be used to refer to WildFly installation directory?

Reply:
Yep. Fixed.